galaxy.yml

```diff
@@ -0,0 +1,56 @@
+### REQUIRED
+
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric characters and underscores. Additionally namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: manala
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: roles
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 0.1.8
+
+# The path to the Markdown (.md) readme file. This path is relative to the root of the collection
+readme: README_COLLECTION.md
```

Comment:
Why not just `README.md` ?

Reply:
Because it's unreadable in our actual format on ansible-galaxy. So we keep the README.md and it's Github format, and introduce an Ansible-galaxy compatible README_COLLECTION.md